inst/examples/app/analytics/app.R

```diff
@@ -18,7 +18,7 @@ library(shiny.telemetry)
 
```

Comment:
```suggestion

# Location to the Log file (tries to use a relative path to instrumentation app)
# and defaults to current directory if that directory does not exist
log_file_path <- file.path("..", "instrumentation", "user_stats.txt")
if (!dir.exists(dirname(log_file_path))) log_file_path <- "user_stats.txt"

```

Reply:
I think this approach with cover all the possible cases.
I'll commit these changes.

Reply:
This change has now been implemented.